spec/unit/generator/entry_spec.rb

```diff
@@ -0,0 +1,351 @@
+# frozen_string_literal: true
+
+describe GitHubChangelogGenerator::Entry do
```

Comment:
Please use the form `RSpec.describe` (not `describe`) for the outermost describe.

Reply:
You can structure tests inside a module, like this:

```ruby
module GitHubChangelogGenerator
  RSpec.describe Entry do
    # ...in here, names will be in the namespace of the module.
    # so that you can use shorter names.
  end
end